network_observability/metrics-alerts-dashboards.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="metrics-dashboards-alerts"]
+= Using metrics, dashboards, and alerts
+include::_attributes/common-attributes.adoc[]
+:context: metrics-dashboards-alerts
+
+toc::[]
+
+The Network Observability Operator uses the `flowlogs-pipeline` to generate metrics out of flow logs. You can make use these metrics by setting custom alerts and viewing dashboards.
+
+include::modules/network-observability-viewing-dashboards.adoc[leveloffset=+1]
+include::modules/network-observability-metrics.adoc[leveloffset=+1]
+
```

Comment:
@jpinsonneau @jotak I think we can put the metrics alerting examples here? Just let me know what you'd like me to show here in terms of includeList examples

Reply:
I've just noticed custom metrics PR is still WIP:
https://github.com/netobserv/network-observability-operator/pull/445/files#diff-f56c4ef1268ddd69d4da318cd3ffc855de718f70c7de5de760fd18ef69f79aeb

@jotak do we still plan to have it as dev preview for 1.5 ?

Reply:
I guess, I was thinking the example here would be includeList metrics and alerts examples. In 1.6 timeframe, the custom metrics can live here as well.

Reply:
@skrthomas for an alerting example I'm suggesting one here: https://github.com/netobserv/documents/pull/62
Yaml is:

```yaml
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  name: netobserv-alerts
spec:
  groups:
  - name: NetObservAlerts
    rules:
    - alert: NetObservIncomingBandwidth
      annotations:
        message: |-
          {{ $labels.job }}: incoming traffic exceeding 10 MBps for 30s on {{ $labels.DstK8S_OwnerType }} {{ $labels.DstK8S_OwnerName }} ({{ $labels.DstK8S_Namespace }}).
        summary: "High incoming traffic."
      expr: sum(rate(netobserv_workload_ingress_bytes_total{SrcK8S_Namespace="openshift-ingress"}[1m])) by (job, DstK8S_Namespace, DstK8S_OwnerName, DstK8S_OwnerType) > 10000000
      for: 30s
      labels:
        severity: warning
```

This is an alert when incoming traffic reaches a given threshold (here 10 MBps) per destination workload

Reply:
Thank you so much :)

Reply:
@jotak would you happen to have an example for includeList? 😇

Reply:
I see the includeList metric, `netobserv_workload_ingress_bytes_total` is in the expression. In the includeList reference, the intro text states: 
> ...configurable in the `spec.processor.metrics.includeList` of the `FlowCollector` custom resource to add or remove metrics.

So, like your comment there says, I think I need to add more context about when a user creates an alert, they specify the includeList metric in the Prometheus rule.

Reply:
This metric I used is present by default so users who didn't configure `includeList` explicitly wouldn't have to worry about it. But I agree, we could say something like: "Make sure the metric `spec.processor.metrics.includeList` either includes `workload_ingress_bytes_total`, or is unset since this metric is enabled by default."

Reply:
I added this as a callout in the YAML: https://github.com/openshift/openshift-docs/pull/69106/files#diff-8d0022591f4f4567d0cc2ff90110d33839696453336a9fd67a8d766ff79f3f48R31